src/common/latest/config.ts

```diff
@@ -1,13 +1,39 @@
-const msBuildVersion = "5.15.1.88158";
-const cliVersion = "4.8.1.3023"; // Has to be the same version as the one embedded in the Scanner for MSBuild
+const msBuildVersion = "6.2.0.85879";
+const cliVersion = "5.0.1.3006"; // Has to be the same version as the one embedded in the Scanner for MSBuild
 
-const scannersLocation = `https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/${msBuildVersion}/`;
-const classicScannerFilename = `sonar-scanner-msbuild-${msBuildVersion}-net46.zip`;
-const dotnetScannerFilename = `sonar-scanner-msbuild-${msBuildVersion}-netcoreapp3.0.zip`;
+// MSBUILD scanner location
+const scannersLocation = `https://github.com/SonarSource/sonar-scanner-msbuild/releases/download/`;
+// CLI scanner location
+const cliUrl = "https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/";
 
-exports.scanner = {
+function getMsBuildClassicFilename(msBuildVersion: string) {
+  return `sonar-scanner-${msBuildVersion}-net-framework.zip`;
+}
+
+function getMsBuildDotnetFilename(msBuildVersion: string) {
+  return `sonar-scanner-${msBuildVersion}-net.zip`;
+}
+
+function getCliScannerFilename(cliVersion: string) {
+  return `sonar-scanner-cli-${cliVersion}.zip`;
+}
+
+export const scanner = {
   msBuildVersion,
   cliVersion,
-  classicUrl: scannersLocation + classicScannerFilename,
-  dotnetUrl: scannersLocation + dotnetScannerFilename,
+
+  cliUrl: `${cliUrl}${getCliScannerFilename(cliVersion)}`,
+
+  msBuildUrlTemplate: (msBuildVersion, isWindows) => {
```

Comment:
(NP) we can add the types here now

```suggestion
  msBuildUrlTemplate: (msBuildVersion: string, isWindows: boolean) => {
```